seed_helen_comments.py

```diff
@@ -0,0 +1,354 @@
+        # color is the inputted rgb value tuples
+
+        r = (r_c - color[0]) ** 2
+        g = (g_c - color[1]) ** 2
+        b = (b_c - color[2]) ** 2
+
+        min_colors[(r + g + b)] = name
+        # print "MIN COLORS", min_colors
+        # print "MIN KEYS ONLY", min_colors[min(min_colors.keys())]
+    return min_colors[min(min_colors.keys())]
+
+
+# def show_colors(final_colors):
+#     # A FXN TO AGGREGATE TOP COLORS BY SHOW
+#     # final_colors is PER image, need to aggregate all final_colors to show colors
+#     show_colors = []
+#     for color in final_colors:
+#         show_colors.append(color)
+
+#     show_colors = final_colors[:6]
+#     return show_colors
+
+
+def get_color_name(color):
+    # pillow errors out if it doesnt have the exact color name to rgb
+
+    # print "at get_color_name"
+    try:
+        closest_name = actual_name = rgb_to_name(color)
+    except ValueError:
+        closest_name = closest_color(color)
+        actual_name = None
+    return closest_name
+
+    # actual_name, closest_name = get_color_name(color)
+
+    # print "closest color name: ", closest_name
+
+
+def get_colors(img, img2):
+    # this function takes in bg & foreground colors and returns top img colors
+    # 1. convert image to palette format
+    img = img.convert('P')
+    img2 = img2.convert('P')
+
+    # 2. convert imgs to RGB for getcolors fxn - returns rgb tuples
+    bg_colors = (img2.convert('RGB').getcolors())
+    foreground_colors = (img.convert('RGB').getcolors())
+
+    # 3. set up our lists to capture top colors for bg & foreground
+    final_bg_colors = []
+    final_colors = []
+    # 4. set up empty list to capture named final colors
+    final_named_colors = []
+
+    named_bg_colors = []
+    named_foreground_colors = []
+
+    # 6. these two loops return lists of named colors
+    for count, rgb in bg_colors:
+        named_bg_colors.append(get_color_name(rgb))
+    for count, rgb in foreground_colors:
+        named_foreground_colors.append(get_color_name(rgb))
+
+    # 7. these create color/count pairs 
+    count_bg_colors = {color: named_bg_colors.count(color) for color in named_bg_colors}
+    count_foreground_colors = {color: named_foreground_colors.count(color) for color in named_foreground_colors}
+
+    # 8. these take the color/count pairs and sort them
+    sorted_bg_colors = sorted(count_foreground_colors.items(), key=lambda x: x[1], reverse=True)[:10]
+    sorted_foreground_colors = sorted(count_foreground_colors.items(), key=lambda x: x[1], reverse=True)[:10]
+
+    # 9. next i have to append top bg colors to final_bg_colors to compare foreground againsr
+    final_bg_colors.append(sorted_bg_colors)
+
+    # 10. next i need to compare the fg colors against bg colors & append to the fg final color list
+
+    for color, count in sorted_foreground_colors:
+        if color not in final_bg_colors:
+            final_colors.append(color)
+        else:
+            pass
```

Comment:
`pass` should really only be used for functions which aren't defined yet.  I would say `return []` or `return None` here so that the caller of the function won't break if its assigning the return value.